### DISCUSSION TITLE
IP/20 not IP/24 for home network, how to enter into scripts?

### DISCUSSION THREAD

**Discussion by mgrimace** [tteck/Proxmox#3070] [Q&A] [answered]
Hi, it's a minor thing but I figured worth asking. When I create a static IP using a script, it asks for the IP in CIDR as IP/24. However, my home network is /20, or 192.168.68.1 and 255.255.240.0. To access the new LXC by SSH (or its webui), I have to manually change the entry from /24 to /20 in PVE/CTID/Network. Is there a way to enter this as /20 in the scripts?

I'm a networking novice and apologies in advance if this is very basic.

**Comment by tteck** [comment, marked as answer]
Use Advanced settings when creating the LXC

**Reply by mgrimace** [reply]
Oh strange, when I did it a moment ago it said that it wasn't a valid CIDR notation, but I just tested again and it was fine. Must be user error, thanks!
